recipes/bobleesj/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set python_min = '3.11' %}
+{%- set name = "bobleesj.utils" -%}
+{%- set version = "0.0.1" -%}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/b/bobleesj_utils/bobleesj_utils-{{ version }}.tar.gz
+  sha256: 469fd9004b472229afa389414cbf49bea986f75b038e714b4b70cff18a4e90f5
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - setuptools-git-versioning >=2.0
+    - pip
+
+  run:
+    - python >={{ python_min }}
+    - numpy
+    - pandas
+    - openpyxl
+
+test: 
+  requires:
+    - pip
+    - pytest
+    - DeepDiff
+    - python {{ python_min }}
+  source_files:
+    - tests
+  commands:
+    - pytest
+
+about:
+  home: https://github.com/bobleesj/bobleesj.utils/
+  summary: Shared utilities for bobleesj packages.
+  license: Free To Use But Restricted
```

Comment:
I'm not a lawyer to read your custom license and say it is OK for us to package and redistributed it. It would be much easier for us if this used a standard OSS OSI approved license.

Reply:
Indeed.. its BSD-3 added!